workflows/python/sdk/order-processor/app.py

```diff
@@ -36,67 +35,27 @@ def main(self):
 
         print("==========Begin the purchase of item:==========", flush=True)
         item_name = default_item_name
-        order_quantity = 10
+        order_quantity = 1
         total_cost = int(order_quantity) * baseInventory[item_name].per_item_cost
         order = OrderPayload(item_name=item_name, quantity=int(order_quantity), total_cost=total_cost)
 
         print(f'Starting order workflow, purchasing {order_quantity} of {item_name}', flush=True)
         instance_id = wfClient.schedule_new_workflow(
             workflow=order_processing_workflow, input=order.to_json())
-        _id = instance_id
-
-        def prompt_for_approval(wfClient: DaprWorkflowClient):
-            """This is a helper function to prompt for approval.
-            Not using the prompt here ACTUALLY, as quickstart validation is required to be automated.
-            
-            But, in case you may want to run this sample manually, you can uncomment the following lines:
-                try:
-                    signal.alarm(15)
-                    approved = input(f'(ID = {_id}) requires approval. Approve? [Y/N] ')
-                    signal.alarm(0) # cancel the alarm
-                except TimeoutError:
-                    approved = "y"
-                if state.runtime_status.name == "COMPLETED":
-                    return
-                if approved.lower() == "y":
-                    wfClient.raise_workflow_event(instance_id=_id, event_name="manager_approval", data={'approval': True})
-                else:
-                    wfClient.raise_workflow_event(instance_id=_id, event_name="manager_approval", data={'approval': False})
-
-                ## Additionally, you would need to import signal and define timeout_error:
-                # import signal
-                # def timeout_error(*_):
-                #     raise TimeoutError
-
-                # signal.signal(signal.SIGALRM, timeout_error)
-            """
-            wfClient.raise_workflow_event(instance_id=_id, event_name="manager_approval", data={'approval': True})
-
-        approval_seeked = False
-        start_time = datetime.now()
-        while True:
-            time_delta = datetime.now() - start_time
-            state = wfClient.get_workflow_state(instance_id=_id)
 
+        try:
```

Comment:
I think we can do `wfClient.wait_for_workflow_start(_id)` here. That's also needed to fix the test failures we had.

Reply:
This is passing now without this! But Marc removed all the calls to `wait_for_external_event` in each of the wfs in order to have the workflows complete during the tests but still want to ensure we leave the logic in there so that users can change it to run it with external inputs.

Reply:
To be clear, the `wail_for_external_event` call is still present in all workflows but due to the orderPayload the workflow logic to request order approval is not run and it goes directly to the process payment activity.